frontend/.yarnrc

```diff
@@ -2,4 +2,4 @@
 # yarn lockfile v1
 
 
-yarn-path ".yarn/releases/yarn-1.22.11.js"
+yarn-path ".yarn/releases/yarn-1.19.0.cjs"
```

Comment:
Had to revert yarn back to 1.19.0 because it started throwing errors anytime I tried to add a package. Even if I would add the package it would just find another one down the list, seems to be a pretty common issue for > 1.19. Still trying to figure out what to do about it.

```
error An unexpected error occurred: "expected workspace package to exist for \"babel-loader\"".
```

Reply:
Ive already reverted the versioning back, TBD if it will continue to throw errors on adding/updating packages